cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -799,7 +799,7 @@ let redirectURIWithParams = "Put the redirect URI with params here"
 
 let credentials = interaction.requestCredential(withAuth: didDocResolution.assertionMethod(), redirectURIWithParams: redirectURIWithParams, opts: nil)
 
-let issuerURI = interaction.issuer().uri() // Optional (but useful)
```

Comment:
Thanks. There's other `issuer().uri()` references in usage.md - would you mind fixing those too?